tools/iana/makefile

```diff
@@ -0,0 +1,5 @@
+
+.PHONY: generate
+generate:
+	@echo "Generating Headers"
+	cd c; ./c_header_coap.py
```

Comment:
This ignores failures from cd, change to `cd c && ./c_header_coap.py`.

Reply:
fixed. Also added more to makefile to use venv now for better python package isolation